【jayj-crm】 [http-nio-8080-exec-3] ERROR com.maker.crm.action.workbench.ClueAction - 线索转换出现异常：Cannot invoke "com.maker.crm.model.User.getId()" because "user" is null
【jayj-crm】 [http-nio-8080-exec-6] ERROR com.maker.crm.action.workbench.ClueAction - 线索转换出现异常：
### Error updating database.  Cause: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'next_contact_time' at row 1
### The error may exist in com/maker/crm/dao/customer/CustomerMapper.xml
### The error may involve com.maker.crm.dao.customer.CustomerMapper.insertCustomer-Inline
### The error occurred while setting parameters
### SQL: insert into tbl_customer (id, owner, name, website,     phone, create_by, create_time, contact_summary,     next_contact_time, description, address     )values(?,?,?,?,?,?,?,?,     ?,?,?)
### Cause: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'next_contact_time' at row 1
; Data truncation: Data too long for column 'next_contact_time' at row 1; nested exception is com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'next_contact_time' at row 1
【jayj-crm】 [http-nio-8080-exec-8] ERROR com.maker.crm.action.workbench.ClueAction - 线索转换出现异常：nested exception is org.apache.ibatis.reflection.ReflectionException: There is no getter for property named 'ContactsId' in 'class com.maker.crm.model.ContactsRemark'
